File: Algorithms/Python/Unique-Paths.py
class Solution(object):

    def uniquePaths(self, m, n):
        """
        :type m: int
        :type n: int
        :rtype: int
        """
        ans = [[0 for x in range(n)] for x in range(m)]
        for i in range(m):
            ans[i][0] = 1
        for j in range(n):
            ans[0][j] = 1
        for i in range(1, m):
            for j in range(1, n):
                ans[i][j] = ans[i - 1][j] + ans[i][j - 1]
        return ans[m - 1][n - 1]

File: Algorithms/Java/Unique Paths.java
public class Solution {
    public int uniquePaths(int m, int n) {
        int[][] num = new int[m][n];
		for (int i = 0; i < m; i++)
			num[i][0] = 1;
		for (int j = 0; j < n; j++)
			num[0][j] = 1;
		for (int i = 1; i < m; i++) {
			for (int j = 1; j < n; j++)
				num[i][j] = num[i - 1][j] + num[i][j - 1];
		}
		return num[m - 1][n - 1];
    }
}